.env.sample

```diff
@@ -1,7 +1,7 @@
 BUGZILLA_API_KEY=
 BUGZILLA_CC_LIST=
 BUGZILLA_HOST=https://bugzilla.allizom.org/
-CELERY_SCHEDULE_INTERVAL=300
+CELERY_SCHEDULE_INTERVAL=30
```

Comment:
Probably don't need to change this here if you're using a separate one for integration tests?

Reply:
TRUE